ckanext/unhcr/click_commands.py

```diff
@@ -5,7 +5,8 @@
 from ckan.plugins import toolkit
 import ckan.model as model
 
-from ckanext.unhcr.models import create_tables, TimeSeriesMetric
+from ckanext.unhcr.commands import expired_users_list, request_renewal
+from ckanext.unhcr.models import AccessRequest, create_tables, TimeSeriesMetric
```

Comment:
```suggestion
from ckanext.unhcr.models import create_tables, TimeSeriesMetric
```

`AccessRequest` is an unused import here